src/annotator/integrations/test/vitalsource-test.js

```diff
@@ -127,7 +127,8 @@ describe('annotator/integrations/vitalsource', () => {
       }, 'Book container element not found');
     });
 
-    it('injects client into content frame', () => {
+    it('injects client into content frame', async () => {
+      await waitFor(() => fakeInjectClient.called);
```

Comment:
Previously `fakeInjectClient` would be called synchronously as a result of the test setup steps. It is now called asynchronously afterwards.